### PR TITLE
Fix sticky issue

### PR DESCRIPTION
**Pull request by qsomazzi**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Testing now.

**Comment by Soullivaneuh** [comment]
Found 3 issues for the moment:

* If you scroll only a little bit, the menu won't become sticky:

sonata_sticky_little_scroll

* Scroll completly at bottom of the page and then, scroll back on top. Menu still sticky, it shouldn't.
* Sometime I got `Uncaught TypeError: Cannot read property 'attr' of undefined` on page load and sticky system is not working at all.

**Comment by qsomazzi** [comment]
What's the size of your 'navbar navbar-static-top' (height) ?

**Comment by Soullivaneuh** [comment]
Exactly `50px` sir. :-)

BTW, I didn't override the navbar at all.

**Comment by qsomazzi** [comment]
same as mine :/ 
mmmmm not cool, will look into it, thanks for the feedback !

**Comment by Soullivaneuh** [comment]
No problem. Ping me if you change something, I'll test it again.

**Comment by OskarStark** [comment]
another issue:
bildschirmfoto 2015-08-26 um 18 03 58

can not click on ```List``` here:
bildschirmfoto 2015-08-26 um 18 05 02

**Comment by qsomazzi** [comment]
@Soullivaneuh done
@OskarStark will look into this

**Comment by OskarStark** [comment]
how is the status @qsomazzi ?

**Comment by qsomazzi** [comment]
Didn't have time to finish yesterday, will look into this on the morning

**Comment by qsomazzi** [comment]
@OskarStark can't reproduce your issue, I think It's fixed with this PR

**Comment by qsomazzi** [comment]
I've added some debounce to optimize Scroll and Resize calls

**Comment by OskarStark** [comment]
@qsomazzi no it works again!

Thank you!

File: Resources/public/Admin.js [hunk 1]
```diff
@@ -535,39 +535,107 @@ var Admin = {
...
+        jQuery(window).scroll(
+            Admin.debounce(function() {
+                if (footer.length && jQuery(window).scrollTop() + jQuery(window).height() == jQuery(document).height()) {
+                    jQuery(footer).removeClass('stuck');
+                }
+
+                if (navbar.length && jQuery(window).scrollTop() === 0) {
+                    jQuery(navbar).removeClass('stuck');
+                }
+            }, 250)
```
Comment (plouc) [inline Resources/public/Admin.js]: missing `;`

**Pull request merged**